src/database/DataTypes/StocktakeItem.js

```diff
@@ -164,11 +164,9 @@ export class StocktakeItem extends Realm.Object {
     });
 
     // Sort (ASC) the options by count, return the first option
```

Comment:
technically this is incorrect?

Reply:
https://github.com/openmsupply/mobile/pull/1008/commits/7aba7b1141240a40701adc9ef9d1fed49dbe85ae

Reply:
https://github.com/openmsupply/mobile/pull/1008/commits/02fa945319c4e7702ecbf2c018645b502a7e754a